### PR TITLE
timescale/timescaledb#932 Fix docker-run.sh

### PR DESCRIPTION
**Pull request by fordred**
This fixed the issues I had using your docker-run.sh script

### CODE REVIEW

File: scripts/docker-run.sh [hunk 1]
```diff
@@ -19,10 +19,11 @@ fi
 docker run -d \
   --name $CONTAINER_NAME $VOLUME_MOUNT \
   -p ${PGPORT}:5432 \
+  -e POSTGRES_PASSWORD=password \
```
Comment (svenklemm) [inline scripts/docker-run.sh]: I don't think hardcoding a password here is the correct solution

Comment (fordred) [reply scripts/docker-run.sh]: Changed to parameter

**Comment by fordred** [comment]
Either fix it or delete it

**Comment by mfreed** [comment]
Adding @RobAtticus to discussion.

**Comment by RobAtticus** [comment]
It appears only to be used in `scripts/start-test-docker.sh`, which in turn is not called from anywhere. Will do a PR.

**Comment by RobAtticus** [comment]
Actually we mention the script file here:
https://docs.timescale.com/v1.1/getting-started/installation/docker/installation-docker#docker

So not sure we want to remove it?

**Comment by mfreed** [comment]
(I think the request was either to potentially fix the script, as suggested above, or remove the script.  I haven't evaluated these changes, just bringing you into the discussion.)

**Comment by RobAtticus** [comment]
I've removed the files in my other PR, so I will close this out.

**Pull request closed without merging**